### ISSUE TITLE
Initial config ideas

### ISSUE THREAD

**Issue by afuggini** [iron-meteor/iron-cli#32]
As we've been discussing in https://github.com/EventedMind/em/issues/6 it would be nice to have an initial / default config when running `em init`. I'm thinking something like Ruby on Rails. Things like:
- A set of popular/frequent packages added by default (Iron Router, Accounts, underscore, etc)
- Hooks for Account (onLogin, onCreateUser) and other
- As well as the type of file to work with (coffee/js, less/css, etc)

Let's discuss.

**Comment by cmather** [comment]
Good idea. Maybe something that goes in .iron/config.json. Perhaps a packages option.

**Comment by lirbank** [comment]
I have a working version of this. See #91 and https://github.com/lirbank/iron-cli/commit/413a36a4edcd239824bdee6f84ed92480cf8e9b9
